scripts/mutatenodes.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -e
+
+SEND_APP_NAME='malleusSender'
+RECV_APP_NAME='malleusReceiver'
+
+usage() {
+    echo "Usage: $0 --hub-url=HUB_URL --num-senders=NUM_SENDERS --send-node-max-sessions=SEND_NODE_MAX_SESSIONS --recv-node-max-sessions=RECV_NODE_MAX_SESSIONS" >&2
+    exit 1
+}
+
+# new arg parsing code that includes default values for the different options.
+for arg in "$@"; do
+    optname=`echo $arg | cut -d= -f1`
+    optvalue=`echo $arg | cut -d= -f2`
+    case $optname in
+        --hub-url) HUB_URL=$optvalue;;
+        --num-senders) NUM_SENDERS=$optvalue;;
+        --send-node-max-sessions) SEND_NODE_MAX_SESSIONS=$optvalue;;
+        --recv-node-max-sessions) RECV_NODE_MAX_SESSIONS=$optvalue;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+if [ -z "$HUB_URL" -o -z "$NUM_SENDERS" -o -z "$SEND_NODE_MAX_SESSIONS" -o -z "$RECV_NODE_MAX_SESSIONS" ]
+then
+    usage
+fi
+
+if [ -z "$JITSI_SSH_CONFIG" ]
```

Comment:
Just a reminder to update this script, If we decide to change anything in the monitor script